Program.cs

```diff
@@ -44,17 +63,31 @@ public static void Main(string[] args)
                         {
                             folderPath += folders[j] + "\\";
                         }
-
+                        folderPath = folderPath.TrimEnd('\\');
                         result.Add(folderPath);
                     }
-
                     return result;
                 })
                 .Distinct()
                 .ToArray();
 
-            UpdateProjectFile(projectFile, files, folderPaths);
-            projectFile.Save(projectFilePath);
+            if (verifyOnly) {
+                if (!CheckProjectFile(projectFile, files, folderPaths)) {
+                    throw new Exception("Project file missing folders or files.");
+                }
+            } else {
+                UpdateProjectFile(projectFile, files, folderPaths);
+                projectFile.Save(projectFilePath);
+            }
+        }
+
+        private static string RemovePrefix(string Test, string Prefix)
```

Comment:
Again violation of C# standard - local variables (and method arguments) should be `camelCase`, not `PascalCase`